packages/sources/css/src/components/skeleton/src/index.css

```diff
@@ -0,0 +1,47 @@
+@import '@vtmn/css-design-tokens/src/typography';
+@import '@vtmn/css-design-tokens/src/opacities';
+@import '@vtmn/css-design-tokens/src/shadows';
+@import '@vtmn/css-design-tokens/src/animations';
+@import '@vtmn/css-design-tokens/src/base-colors';
+@import '@vtmn/css-design-tokens/src/themes/default';
+
+.vtmn-skeleton {
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  overflow: hidden;
+  background-color: var(--vtmn-semantic-color_background-tertiary);
+}
+.vtmn-skeleton::after {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  transform: translateX(-100%);
+  background-image: linear-gradient(
+    90deg,
+    rgba(255, 255, 255, 0) 0,
+    rgba(255, 255, 255, 0.3) 10%,
+    rgba(255, 255, 255, 0.6) 40%,
+    rgba(255, 255, 255, 0.6) 60%,
+    rgba(255, 255, 255, 0.3) 90%,
+    rgba(255, 255, 255, 0)
+  );
+  animation: shimmer 1.5s infinite;
+  content: '';
+}
+.vtmn-skeleton_line {
+  height: 8px;
+  border-radius: 8px;
+}
+.vtmn-skeleton_avatar {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+}
```

Comment:
Should we not use rem() function for sizing element ?

Reply:
You're right, done!